app/scripts/modules/core/src/presentation/icons/vectors/mdConstraintDependsOn.svg

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 22.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="-1 0 24 24" style="enable-background:new 0 0 24 24;" xml:space="preserve">
```

Comment:
@gcomstock I hand-modified the `viewBox` on this to nudge the paths over to the right by... 1/24th of the canvas I guess 🤷‍♂🕵 

While it was technically centered on its 24x24 canvas and sized properly to fill as much of the space as it could, _visually_ it looked way off-center because of the shape and visual weight of the paths. I couldn't figure out what was driving me crazy about it until I went in and started poking at the SVG to move it around, but doing this _really_ pulled the alignment back together in the list of cards. Here's a direct comparison of original/nudged, if you agree this is the right call can you adjust the original SVG to sit in this spot on the canvas so we don't have to worry about losing my `viewBox` hack with it gets modified in the future?

**Without nudge:**
<img width="878" alt="without-nudge" src="https://user-images.githubusercontent.com/1850998/78505254-249e3780-7727-11ea-85b5-63cdb6d344dd.png">

**With nudge:**
<img width="882" alt="with-nudge" src="https://user-images.githubusercontent.com/1850998/78505262-2d8f0900-7727-11ea-8c04-68fe7465297c.png">

Reply:
Good call!